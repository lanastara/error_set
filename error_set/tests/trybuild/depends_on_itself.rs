use error_set::error_set;

error_set! {
    BookParsingError = {
        MissingDescriptionArg
    } || BookParsingError;
    BookSectionParsingError = {
        MissingNameArg,
        NoContents,
    };
}