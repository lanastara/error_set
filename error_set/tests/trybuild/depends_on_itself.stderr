error: Cycle Detected: BookParsingError->BookParsingError
 --> tests/trybuild/depends_on_itself.rs:4:5
  |
4 |     BookParsingError = {
  |     ^^^^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/trybuild/depends_on_itself.rs:11:2
   |
11 | }
   |  ^ consider adding a `main` function to `$DIR/tests/trybuild/depends_on_itself.rs`
